flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -91,7 +89,7 @@ public Builder env(StreamExecutionEnvironment newEnv) {
     }
 
     public Builder filters(List<Expression> filters) {
```

Comment:
I have been wondering if really need to duplicate every ScanContext/Builder API in the FlinkSource. Should we just let FlinkSource builder take the ScanContext, which is constructed separately before the FlinkSource?

Reply:
We've discussed this before in. https://github.com/apache/iceberg/pull/1346#issuecomment-699056581 and https://github.com/apache/iceberg/pull/1509#issuecomment-699778221.   The conclusion is :  exposing only one simple `FlinkSource`  API to end users , rather than expose two builders.     That make sense to me,  flink data stream developers will find it's easier to construct the streaming source, though we iceberg developers need to write those duplicated setters .